easybuild/easyconfigs/r/Reapr/Reapr-1.0.18-foss-2019a-Perl-5.28.1.eb

```diff
@@ -0,0 +1,49 @@
+easyblock = 'MakeCp'
+
+name = 'Reapr'
+version = '1.0.18'
+versionsuffix = '-Perl-%(perlver)s'
```

Comment:
@hrnagel Please remove the `versionsuffix` (and rename the easyconfig accordingly), there's no need for it since we stick to a single `Perl` version per "generation" of easyconfigs (e.g. `2019a` in this case)

Reply:
Done